.github/workflows/ci.yml

```diff
@@ -67,7 +67,8 @@ jobs:
     - name: Run browser-based end-to-end tests (Linux)
       # TODO: Add Edge/merge with Windows setup once Edge is available on Linux:
       # Connections are flakier in CI, so retry failed network requests with --retry-test-pages.
-      run: npm run e2e-test-browser -- firefox:headless,chrome:headless --retry-test-pages --hostname localhost
+      # Setting a test-specific user agent is only possible for Chrome: https://stackoverflow.com/a/59358925
+      run: npm run e2e-test-browser -- "firefox:headless,chrome:headless:userAgent='Browser-based solid-client end-to-end tests running in CI. Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) $(chrome --version) Safari/537.36'" --retry-test-pages --hostname localhost
```

Comment:
Just curious - why add the `--hostname localhost` option...?

Reply:
Good question, should've been added to the comment I suppose, cause I had to look it up too :) Looks like it's required with `--retry-test-pages`: https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#retrytestpages

I'll try to remember adding that later today.

Reply:
03c7f48c61d26e050df5010168999bfa52934cbf